application/common/components/passwordStore/Ldap.php

```diff
@@ -70,11 +70,8 @@ class Ldap extends Component implements PasswordStoreInterface
      */
     public $updateAttributesOnSetPassword = [];
 
-    /** @var \Adldap\Connections\Provider */
-    public $ldapProvider;
-
-    /** @var \Adldap\Adldap LDAP client*/
-    public $ldapClient;
+    /** @var \LdapRecord\Connection */
+    public Connection $ldapConnection;
```

Comment:
To keep lines changed at a minimum, I could keep the `$ldapClient` name instead of this if desired,

Reply:
That might be preferred, but not worth changing back at this point.